src/main/java/com/fvv/bookstore/view/OrderViewImpl.java

```diff
@@ -94,6 +95,109 @@ public void addOrder() {
+			Integer yearSearch = Integer.parseInt(JOptionPane.showInputDialog("Insert the year to search"));
+			List<Order> orders = this.orderController.listTotalOrdersMonth(monthSearch, yearSearch);
+			if(orders != null && !orders.isEmpty()) {
+				Integer quantity = this.orderController.calculateQtyOrders(orders);
+				Double amount = this.orderController.calculateTotalOrders(orders);
+				sb.append("Quantity of orders: ").append(quantity).append(" - Amount in the month: R$").append(amount).append(Constants.LINE_SEPARATOR)
+				.append(Constants.LINE_SEPARATOR);
+				for(Order ord : orders) {
+					sb.append("Date: ").append(ord.getCreationDate()).append(" - Order ID: ").append(ord.getId()).
+					append(" - Order Amount: R$").append(ord.getOrderAmount()).append(Constants.LINE_SEPARATOR).append("Employee ID: ").
+					append(ord.getEmployee().getId()).append(" - Name: ").append(ord.getEmployee().getName()).append(", Customer ID: ").
+					append(ord.getCustomer().getId()).append(" - Name: ").append(ord.getCustomer().getName()).append(Constants.LINE_SEPARATOR).
+					append(Constants.LINE_SEPARATOR);
+				}
+				JOptionPane.showMessageDialog(null, sb.toString(), "Listing the Orders of the Searched Date: " + yearSearch + "-" + monthSearch, 
+						JOptionPane.PLAIN_MESSAGE);
+			} else {
+				JOptionPane.showMessageDialog(null, "There are no items to show!");
+			}			
+		} catch (ControllerException | OrderNotFoundException e) {
+			e.printStackTrace();
+			JOptionPane.showMessageDialog(null, e.getMessage(), "Error", 
+					JOptionPane.ERROR_MESSAGE);
+		}
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void listTotalSalesPerSeller() {
+		try {
+			StringBuilder sb = new StringBuilder();
+			Long idSearch = Long.parseLong(JOptionPane.showInputDialog("Insert the employee ID to search"));
+			Employee employee = this.employeeController.findEmployee(idSearch);
+			List<Order> orders = this.orderController.listTotalSalesPerSeller(employee);
+			
+			if(orders != null && !orders.isEmpty()) {
+				Integer quantity = this.orderController.calculateQtyOrders(orders);
+				Double amount = this.orderController.calculateTotalOrders(orders);
+				sb.append("Employee ID: ").append(employee.getId()).append(" - Name: ").append(employee.getName()).append(Constants.LINE_SEPARATOR)
+				.append("Quantity of orders: ").append(quantity).append(" - Amount in the period: R$").append(amount).append(Constants.LINE_SEPARATOR)
+				.append(Constants.LINE_SEPARATOR);
+				
+				for(Order ord : orders) {
+					sb.append("Order ID: ").append(ord.getId()).append(" - Order Amount: R$").append(ord.getOrderAmount()).append(", Customer ID: ")
+					.append(ord.getCustomer().getId()).append(" - Name: ").append(ord.getCustomer().getName()).append(Constants.LINE_SEPARATOR)
+					.append(Constants.LINE_SEPARATOR);
+				}
+				
+				JOptionPane.showMessageDialog(null, sb.toString(), "Listing the Orders of the Employee Searched: " + idSearch, 
```

Comment:
Remove all "ID" words in the report, it makes it clearer

Reply:
ok